Makefile

```diff
@@ -69,9 +69,9 @@ tools-linux-amd64:
 $(foreach SERVICE,$(SERVICES),$(eval $(SERVICE_RULES)))
 $(foreach TOOL,$(TOOLS),$(eval $(TOOL_RULES)))
 
-install-vendor: .gitmodules
-	@echo Updating submodules
-	git submodule update --init --recursive
+install-vendor: install-glide
```

Comment:
Probably also need to fix up `license_dir` in the Makefile to point to the new path under .ci